data/pathfinder/paizo/player_companion/adventurers_armory_2/aa2_equipmods.lst

```diff
@@ -23,7 +23,7 @@ Vitalguard					KEY:Special Quality ~ Vitalguard ~ Armor							TYPE:Armor
 # Special Materials
 # ==============================
 
-Singing Steel				KEY:Material ~ Singing Steel ~ Weapon							TYPE:BaseMaterial.MasterworkQuality.Ammunition				COST:120	VISIBLE:QUALIFY	ITYPE:Metal.Singing Steel.Masterwork	REPLACES:Special Quality ~ Masterwork ~ Armor	SOURCEPAGE:p.28	PRETYPE:1,Metal																																																																																																																											BONUS:WEAPON|DAMAGE|-1|TYPE=Enhancement.STACK|PRETYPE:1,PIERCING,SLASHING	SPROP:20 hp/inch, hardness 10
+Singing Steel				KEY:Material ~ Singing Steel ~ Ammunition							TYPE:BaseMaterial.MasterworkQuality.Ammunition				COST:120	VISIBLE:QUALIFY	ITYPE:Metal.Singing Steel.Masterwork	REPLACES:Special Quality ~ Masterwork ~ Armor	SOURCEPAGE:p.28	PRETYPE:1,Metal																																																																																																																											BONUS:WEAPON|DAMAGE|-1|TYPE=Enhancement.STACK|PRETYPE:1,PIERCING,SLASHING	SPROP:20 hp/inch, hardness 10
```

Comment:
Looks like the replaces needs to be fixed as well. Is this something you can address?

Reply:
Can do